android/app/src/main/java/io/peggyjo/peggymeter/HistoryTextFragment.java

```diff
@@ -31,6 +38,37 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
         dataController.addMoodListener(this);
         // Inflate the layout for this fragment
         View view = inflater.inflate(R.layout.fragment_history_text, container, false);
+        ListView table = view.findViewById(R.id.LOG_ENTRIES_CONTAINER);
+        table.setOnItemLongClickListener(new AdapterView.OnItemLongClickListener() {
+            @Override
+            public boolean onItemLongClick(AdapterView<?> adapterView, View view, int i, long l) {
+                LogEntry selectedEntry = (LogEntry)table.getItemAtPosition(i); //used instead of getSelectedItem because getSelectedItem causes a null pointer exception
```

Comment:
Can you move the comment one line up please?
I know this codebase was done in haste but let's try to follow https://source.android.com/setup/contribute/code-style from now on.
So max line length = 100 chars.

Reply:
I will remember that.